src/app/app.component.ts

```diff
@@ -45,4 +42,43 @@ export class AppComponent implements OnInit {
             console.error(ex);
         }
     }
+
+    private replacedTitles(url: string): string {
```

Comment:
Instead of using If statements, this is good place to use switch statements.
link to ref: https://www.geeksforgeeks.org/switch-vs-else/

Reply:
i tired switch statements it would not work because of condition

Reply:
I can create other PR applying a fix for this. and see why it didn't work for you :)

Reply:
also have been asking around for a better solution like mat and andnan but nobody couldn't come up with something better... i know it looks ugly...

Reply:
that would be great